pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computesharedvpcserviceproject/dependencies.yaml

```diff
@@ -17,45 +17,53 @@ kind: Project
 metadata:
   labels:
     label-one: "value-one"
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: "abandon"
   name: vpchost-${uniqueId}
 spec:
+  resourceID: ${TEST_DEPENDENT_FOLDER_PROJECT_ID}
+  name: "VPC Host Dependent Project"
```

Comment:
I thought every pre-created project will have name `Dependent Project`? If there are exceptions, could you ensure the exceptions are documented as the comment of the internal shared vars?

Reply:
Added comment